### PR TITLE
🎉 New source connector: Glassfrog

### PR DESCRIPTION
**Pull request by rparrapy**
## What
Add new native source connector for Glassfrog

## How
Native connector using the Python CDK, pulls all resources available through the [Glassfrog API](https://documenter.getpostman.com/view/1014385/glassfrog-api-v3/2SJViY).

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
No breaking changes.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

## Tests

<details><summary><strong>Acceptance</strong></summary>

Screen Shot 2022-06-16 at 21 36 19

</details>

### CODE REVIEW

**Comment by alafanechere** [comment]
Thank you @rparrapy for this contribution. We need a sandbox account for Glassfrog to run the acceptance tests in our CI. I created an issue so that our team can create it. We'll review it once we have this account.

File: airbyte-integrations/connectors/source-glassfrog/source_glassfrog/source.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        return None
+
+    def request_params(
+        self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> MutableMapping[str, Any]:
+        """
+        TODO: Override this method to define any query parameters to be set. Remove this method if you don't need to define request params.
+        Usually contains common params e.g. pagination size etc.
+        """
+        return {}
```
Comment (harshithmullapudi) [inline airbyte-integrations/connectors/source-glassfrog/source_glassfrog/source.py:31]: Can we remove this method?

File: airbyte-integrations/connectors/source-glassfrog/source_glassfrog/source.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        json_response = response.json()
+        records = json_response.get(self.data_field, []) if self.data_field is not None else json_response
+
+        for record in records:
+            yield record
+
+
+class Assignments(GlassfrogStream):
+    data_field = "assignments"
```
Comment (harshithmullapudi) [inline airbyte-integrations/connectors/source-glassfrog/source_glassfrog/source.py:42]: Could you add a link to the documentation here? and same for other streams also

**Review by harshithmullapudi — CHANGES_REQUESTED** [review]

**Pull request merged**